stack.yaml

```diff
@@ -96,6 +96,7 @@ nix:
   shell-file: shell.nix
 
 extra-deps:
+- transformers-0.5.5.0
```

Comment:
(nitpick) let's add a comment why version override is needed here

Reply:
Added appropriate link to transformers issue.